tools/schemas/schema.ts

```diff
@@ -65,3 +65,9 @@ export const SourceUrlsSchema = z
     $schema: z.string(),
   })
   .catchall(z.record(z.string(), z.string().url()));
+
+export const AbandonmentsSchema = z
```

Comment:
I think we should add a test which validates the file against this Schema?

Reply:
In fact, I added this schema only because we have such a test 🙂

Reply:
https://github.com/renovatebot/renovate/blob/main/test/other/validate-schemas.spec.ts